warehouse/models/mart/transit_database/dim_properties_and_features.sql

```diff
@@ -1,7 +1,10 @@
 {{ config(materialized='table') }}
 
-WITH stg_transit_database__properties_and_features AS (
-    SELECT * FROM {{ ref('stg_transit_database__properties_and_features') }}
+WITH latest AS (
+    {{ get_latest_dense_rank(
```

Comment:
do these not need a partition-by?

Reply:
Nope! I added the partition-by in this PR to support the get-latest-by-date operation where you partition by date and get the latest timestamp. But because we do that operation in the staging table, we just need to get everything from the latest date in the mart tables. 

But this did make me realize that time is just a string, I will need to cast it to make sure that sorting is correect, will do that